lib/hammer_cli_foreman/template.rb

```diff
@@ -142,6 +142,36 @@ class DeleteCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class ExportCommand < HammerCLIForeman::DownloadCommand
+      command_name    "export"
+      action :export
+
+      def default_filename
+        "Template-#{Time.new.strftime("%Y-%m-%d")}.txt"
+      end
+
+      build_options
+
+      def execute
+        response = send_request
+        if response.code == 204
+          print_message(_('The template is not ready yet.'))
```

Comment:
This is redundant, the endpoint won't answer `204` at least currently: https://github.com/theforeman/foreman/blob/develop/app/controllers/api/v2/provisioning_templates_controller.rb#L129